src/components/addQuestion/index.js

```diff
@@ -0,0 +1,69 @@
+import { connect } from 'react-redux';
+import { createQuestion } from '../../store/question.js';
+
+import './index.css';
+
+let AddQuestionForm = ({ onSave, positionClass }) => {
+	const onSubmit = (event) => {
+		event.preventDefault();
+		const {
+			question: { value: question },
+			askee: { value: askee },
+			status: { value: status },
+		} = event.target;
+		onSave({ question, askee, status });
+	};
+	return (
+		<div className={positionClass}>

```

Comment:
Use the prettier default (2 spaces) for indentation. This will help tremendously in code reviews (you can see here how much the tabs push everything and make it wrap awkwardly).